### PR TITLE
Image conversion flow with sharp

### PR DESCRIPTION
**Pull request by rohitsanj**
Added a dropdown and connected the convert button to sharp conversion function.

### CODE REVIEW

**Comment by radiantly** [comment]
`sharp` returns an error and does not run on Linux. It looks like an issue was opened in the sharp repository https://github.com/lovell/sharp/issues/2356 and was fixed upstream, but has not been released yet.

In the meantime, will test on Windows.

File: renderer/style.css [hunk 1]
```diff
@@ -13,23 +13,27 @@ body,
 
 /* File picker (Main screen) */
 .file-picker {
+  width: 50%;
+  height: 50%;
```
Comment (radiantly) [inline renderer/style.css:17]: Wait, why is this changed to 50%? The "Choose files" text is no longer in the center now.

**Review by radiantly — APPROVED** [review]

**Review by chebro — APPROVED** [review]
